docs/src/pages/system/advanced/advanced.md

```diff
@@ -0,0 +1,21 @@
+# Advanced
+
+<p class="description">Here you can find examples of how you can use the system on your custom components.</p>
+
+## Adding the `sx` prop on your custom components
+
+The easiest way to support the `sx` prop on your custom components is to use the `experimentalStyled()` utility that comes from `@material-ui/core/styles` for creating your custom components. Here is an example of how it can be use:
```

Comment:
```suggestion
The easiest way to support the `sx` prop on your custom components is to use the `experimentalStyled()` utility that comes from `@material-ui/core/styles` for creating your custom components. Here is an example of how it can be used:
```